frontend/public/style/_overrides.scss

```diff
@@ -191,3 +191,7 @@ tags-input .autocomplete .suggestion-item em {
 .modal-footer .alert {
   text-align: left;
 }
+
+.ReactVirtualized__List:focus {
+  outline: none;
```

Comment:
We probably want `tabIndex={null}` instead. If we want the element to have a tab index, we should leave the outline.

Reply:
I notice we remove outline in many places in the UI, which makes things really confusing if you tab through using the keyboard because focus disappears.